application/src/main/java/org/opentripplanner/service/streetdecorator/internal/DefaultOsmStreetDecoratorRepository.java

```diff
@@ -0,0 +1,41 @@
+package org.opentripplanner.service.streetdecorator.internal;
+
+import jakarta.inject.Inject;
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Optional;
+import org.opentripplanner.service.streetdecorator.OsmStreetDecoratorRepository;
+import org.opentripplanner.service.streetdecorator.model.EdgeLevelInfo;
+import org.opentripplanner.street.model.edge.Edge;
+import org.opentripplanner.utils.tostring.ToStringBuilder;
+
+public class DefaultOsmStreetDecoratorRepository
```

Comment:
Add javadoc.

Reply:
The javadoc can be found in the interface. This is the way it was done in the `osminfo` package as well